atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -345,7 +345,7 @@ def get_logs(self):
             logfile = NamedTemporaryFile(prefix=self.build_id,
                                          suffix=".log",
                                          mode='wb')
-            logfile.write(logs.encode('UTF-8'))
+            logfile.write(logs)
```

Comment:
Hmm, right. Not sure how to do that correctly - use `six` find out the string type?